src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/VisitSchedulerExceptionHandler.kt

```diff
@@ -74,6 +75,18 @@ class VisitSchedulerExceptionHandler(
     return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(error)
   }
 
+  @ExceptionHandler(MigratedVisitCapacityGroupMatchException::class)
+  fun handleMigratedVisitCapacityGroupMatchException(e: Exception): ResponseEntity<ErrorResponse> {
+    log.debug("Migration exception: {}", e.message)
+    val error = ErrorResponse(
+      status = HttpStatus.BAD_REQUEST,
+      userMessage = "Migration failure: could not find matching session template",
```

Comment:
Is this message correct? The handler talks about visit capacity group?

Reply:
just generic message to customer, dev message will have the detail

Reply:
generic message to user, dev message has detail